src/components/App/App.js

```diff
@@ -0,0 +1,24 @@
+import React from 'react';
+import { Provider } from 'react-redux';
+import PropTypes from 'prop-types';
+
+import AppRouter from '../../router/AppRouter';
+import storeShape from '../../prop-types/storeShape';
+
+const App = ({ store }) => (
+  <div>
+    <Provider store={store}>
+      <AppRouter />
+    </Provider>
+  </div>
+);
+
+App.propTypes = {
+  store: PropTypes.shape(storeShape),
```

Comment:
do you have some arguments to pass the store as a props instead of initialize it here, in this file ? :)